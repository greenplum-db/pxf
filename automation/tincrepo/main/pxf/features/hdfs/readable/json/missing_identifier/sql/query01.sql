-- @description query01 for PXF HDFS Readable Json with missing identifier test cases

SELECT * from jsontest_missing_identifier ORDER BY id;

SELECT * from jsontest_missing_identifier_filefrag ORDER BY id;