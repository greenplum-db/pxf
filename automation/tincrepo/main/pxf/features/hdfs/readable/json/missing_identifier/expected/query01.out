-- @description query01 for PXF HDFS Readable Json with missing identifier test cases
SELECT * from jsontest_missing_identifier ORDER BY id;
           created_at           |         id         | text  | user.screen_name | entities.hashtags[0] | coordinates.coordinates[0] | coordinates.coordinates[1] 
--------------------------------+--------------------+-------+------------------+----------------------+----------------------------+----------------------------
 Fri Jun 07 22:45:02 +0000 2013 | 343136547115253761 | text1 | SpreadButter     | tweetCongress        |                            |                           
 Fri Jun 07 22:45:02 +0000 2013 | 343136547123646465 | text2 | patronusdeadly   |                      |                            |                           
 Fri Jun 07 22:45:02 +0000 2013 | 343136547136233472 | text3 | NoSecrets_Vagas  |                      |                            |                           
(3 rows)

SELECT * from jsontest_missing_identifier_filefrag ORDER BY id;
           created_at           |         id         | text  | user.screen_name | entities.hashtags[0] | coordinates.coordinates[0] | coordinates.coordinates[1] 
--------------------------------+--------------------+-------+------------------+----------------------+----------------------------+----------------------------
 Fri Jun 07 22:45:02 +0000 2013 | 343136547115253761 | text1 | SpreadButter     | tweetCongress        |                            |                           
 Fri Jun 07 22:45:02 +0000 2013 | 343136547123646465 | text2 | patronusdeadly   |                      |                            |                           
 Fri Jun 07 22:45:02 +0000 2013 | 343136547136233472 | text3 | NoSecrets_Vagas  |                      |                            |                           
(3 rows)

